lib/unix.c

```diff
@@ -138,7 +151,7 @@ qb_sys_mmap_file_open(char *path, const char *file, size_t bytes,
 		}
 		free(buffer);
 	}
-
+#endif
```

Comment:
As discovered by @wladmis , a newline was lost unfortunately.

```suggestion
#endif

```